src/foundry/client/pixi/extensions/polygon.d.mts

```diff
@@ -147,6 +141,6 @@ declare module "pixi.js" {
      * @param options - Options which configure how the intersection is computed
      * @returns The intersected polygon
      */
-    intersectRectangle(rect: PIXI.Rectangle, options?: Record<string, unknown>): PIXI.Polygon;
+    intersectRectangle(rect: PIXI.Rectangle, options?: Parameters<PIXI.Rectangle["intersectPolygon"]>[1]): PIXI.Polygon;
```

Comment:
Same nit about adding it to the namespace for reusability if you think that's valuable.